back/sssscs/src/main/java/com/ib/util/security/JwtRequestFilter.java

```diff
@@ -64,15 +77,14 @@ public UsernamePasswordAuthenticationToken getAuthFromToken(String jwt) {
 		try {
 			userDetails = this.getUserDetailsFromJwtToken(jwt);
 		} catch (JwtException | IllegalArgumentException | UsernameNotFoundException e) {
-			return null;
+			throw e;
```

Comment:
We don't really need to catch this anymore, just let it propagate to caller. Maybe add a `throws` keyword to function

Reply:
[664aed0](https://github.com/magley/sssscs/pull/40/commits/664aed0655d54223b695412ce680c1c49b469929)